src/test/java/org/jenkinsci/plugins/mesos/MesosCloudDescriptorTest.java

```diff
@@ -0,0 +1,63 @@
+package org.jenkinsci.plugins.mesos;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+
+import hudson.util.FormValidation.Kind;
+import org.jenkinsci.plugins.mesos.MesosCloud.DescriptorImpl;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+@ExtendWith(TestUtils.JenkinsParameterResolver.class)
+public class MesosCloudDescriptorTest {
+
+  @Test
+  void validateMesosMasterUrl(TestUtils.JenkinsRule j) throws Exception {
+    MesosCloud.DescriptorImpl descriptor = new DescriptorImpl();
+    assertThat(descriptor.doCheckMesosMasterUrl("http/other").kind, is(Kind.ERROR));
+    assertThat(descriptor.doCheckMesosMasterUrl("zk://localhost:5050").kind, is(Kind.ERROR));
```

Comment:
https://github.com/mesosphere/dcos-jenkins-service/blob/master/conf/jenkins/config.xml#L22 we currently read the mesos master url from ZK, this is the config the jenkins master will have

Reply:
I don't think our V1 Mesos client supports this. But we use a URI: https://github.com/mesosphere/usi/blob/master/mesos-client/src/main/scala/com/mesosphere/mesos/client/MesosClient.scala#L304. I make the change.

Reply:
Ok, I did a quick dive. It's a little bit more complicated since our client does not support HTTPS either. Could we do a follow up and not block this PR? Once we agree on the Mesos Client side we can adapt the validation.

Reply:
ya sounds good to me